src/features/presence/__tests__/PresenceCollaboratorsList.test.js

```diff
@@ -132,56 +132,7 @@ describe('features/presence/PresenceCollaboratorsList', () => {
         });
     });
 
-    describe('renderTimestampMessage()', () => {
-        test('should return null when interactionType is an unkown type', () => {
-            const wrapper = shallow(<PresenceCollaboratorsList intl={intl} collaborators={collaboratorList} />, {
-                disableLifecycleMethods: true,
-            });
-            const instance = wrapper.instance();
-            const res = instance.renderTimestampMessage(123, 'test1234');
-            expect(res).toEqual(null);
-        });
-
-        test('should not return null when interactionType is a known type', () => {
-            const wrapper = shallow(<PresenceCollaboratorsList intl={intl} collaborators={collaboratorList} />, {
-                disableLifecycleMethods: true,
-            });
-            const instance = wrapper.instance();
-            const res = instance.renderTimestampMessage(123, 'user.item_preview');
-            expect(res).not.toEqual(null);
-        });
-    });
-
     describe('render()', () => {
-        test('should correctly render an empty PresenceCollaboratorsList', () => {
-            const collaborators = [];
-
-            const wrapper = shallow(<PresenceCollaboratorsList intl={intl} collaborators={collaborators} />, {
```

Comment:
Can we add a test to ensure the collaborator items are output correctly in the DOM?